frontend/public/components/utils/resource-log.jsx

```diff
@@ -44,6 +48,20 @@ const LogControls = ({dropdown, onDownload, toggleFullscreen, isFullscreen, stat
       {dropdown && <div className="co-toolbar__item">{dropdown}</div>}
     </div>
     <div className="co-toolbar__group co-toolbar__group--right">
+      {!_.isEmpty(podLogLinks) && _.map(podLogLinks, (link, i) => {
+        const compiled = _.template(link.spec.hrefTemplate);
```

Comment:
I bet we might want to wrap this in a `_.attempt(_.template(link.spec.hrefTemplate))`... people might pass in a bad template that doesn't work.  I don't know what lodash does if there is a mismatch in `{{}}` or other things.